service/dap/command.go

```diff
@@ -113,3 +120,16 @@ func (s *Session) evaluateConfig(_, _ int, expr string) (string, error) {
 		return res, nil
 	}
 }
+
+func (s *Session) sources(_, _ int, args string) (string, error) {
+	sources, err := s.debugger.Sources(args)
+	if err != nil {
+		return "", err
+	}
+	sort.Strings(sources)
```

Comment:
rpc client doesn't see to do any sorting. Isn't the data coming from debugger sorted?

Reply:
the terminal client sorts the results before printing the list of source files.

Reply:
I see. Then, yeah, what @aarzilli said: we should consider sharing code with the terminal package at some point.